Shared/Samples/Display content of utility network container/DisplayContentOfUtilityNetworkContainerView.Model.swift

```diff
@@ -29,7 +29,7 @@ extension DisplayContentOfUtilityNetworkContainerView {
         @Published private(set) var legendItems: [LegendItem] = []
         
         /// The Naperville Electric Containers web map.
-        let map = Map(item: .napervilleElectricalNetwork())
+        let map = Map(item: PortalItem.napervilleElectricalNetwork())
```

Comment:
I'm curious why you changed the type being extended to `PortalItem`? In my opinion being able to use an implicit member expression here is preferred. My suggestion would be to undo those changes.

Similar elsewhere.